migration/migration/012-tracing.sql

```diff
@@ -66,30 +66,8 @@ GRANT SELECT ON TABLE _ps_trace.tag TO prom_reader;
 GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE _ps_trace.tag TO prom_writer;
 GRANT USAGE ON SEQUENCE _ps_trace.tag_id_seq TO prom_writer;
 
--- create the partitions of the tag table
-DO $block$
-DECLARE
-    _i bigint;
-    _max bigint = 64;
-BEGIN
-    FOR _i IN 1.._max
-    LOOP
-        EXECUTE format($sql$
-            CREATE TABLE _ps_trace.tag_%s PARTITION OF _ps_trace.tag FOR VALUES WITH (MODULUS %s, REMAINDER %s)
-            $sql$, _i, _max, _i - 1);
-        EXECUTE format($sql$
-            ALTER TABLE _ps_trace.tag_%s ADD PRIMARY KEY (id)
-            $sql$, _i);
-        EXECUTE format($sql$
-            GRANT SELECT ON TABLE _ps_trace.tag_%s TO prom_reader
-            $sql$, _i);
-        EXECUTE format($sql$
-            GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE _ps_trace.tag_%s TO prom_writer
-            $sql$, _i);
-    END LOOP;
-END
-$block$
-;
+-- we used to create a bunch of partitions here, but since we drop
```

Comment:
I don't think that we should be retroactively modifying the migration scripts. If we do, we lose the ability to reason about what was present when. I might be in the minority here though.

Reply:
agreed. `012-tracing.sql` has already been released into the wild. Granted our new migration tracking system hasn't, but I THINK that since the 0.0.0 script prepopulates the migration table and includes 12, this migration will probably get skipped.

Reply:
My point is that it does a lot of useless work when it's not skipped, but I won't insist on having it.